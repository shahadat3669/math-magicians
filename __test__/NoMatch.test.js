/**
 * @jest-environment jsdom
 * @jest-environment-options {"url": "https://jestjs.io/"}
 */

import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NoMatch from '../src/components/NoMatch';

describe('Components render correctly', () => {
  it('NoMatch component', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NoMatch />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
