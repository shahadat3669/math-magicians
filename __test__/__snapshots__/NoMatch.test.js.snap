// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components render correctly NoMatch component 1`] = `
<div>
  <h2>
    Nothing to see here!
  </h2>
  <p>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to the home page
    </a>
  </p>
</div>
`;
